shell/client/shell.html

```diff
@@ -41,6 +41,9 @@
   </div>
   {{else}}
   <div id="topbar-right">
+    {{#if isDemoUser}}
+    <a id="alert" href="https://sandstorm.io/install/">Run your own »</a>
```

Comment:
Thanks for asking! I just added a link to
https://github.com/sandstorm-io/sandstorm-website/pull/69 in the pull
request description, which contains proposed contents for /install/